acis_thermal_check/__init__.py

```diff
@@ -1,4 +1,6 @@
-__version__ = "2.9.0"
```

Comment:
@jzuhone As an aside, I note that if ACIS has any code that is importing the modules to check version, I expect with with setuptools-scm the 'in-the-code' versions will now have your 'v' prefix if you continue to make github releases with it (like 'v2.9.0').

Reply:
We plan on dropping the "v" prefix from now on. Thanks for the heads-up, @jeanconn.